TabloidMVC/Views/Post/Index.cshtml

```diff
@@ -11,6 +11,9 @@
     <p>
         <a class="btn btn-primary" asp-action="Create">New Post</a>
     </p>
+    <p>
+        <a class="btn btn-primary" asp-action="Create">My Posts</a>
```

Comment:
We can let this go for now, because I know you're working on this in another ticket, but technically the "My Posts" button shouldn't be on this PR, especially if the button goes somewhere unexpected (i.e. the create form)

Reply:
Yep, you got me. I had added that button today while starting the next ticket, and then I realized I needed to do the pull request for the other ticket first! I'll keep it in mind from now on